.travis.yml

```diff
@@ -9,8 +9,8 @@ jobs:
   include:
     - stage: run tests
       script:
-        # Ensures `package.json` and `yarn.lock` stay in sync
-        - yarn check --integrity
+        - set -e
```

Comment:
Could do with a comment. 'Ensure Travis aborts without running tests if the lockfile needs updating'?